luda-editor/luda-editor-client/src/app/sequence_list/mod.rs

```diff
@@ -1,35 +1,48 @@
 use self::{
     events::SequenceListEvent,
-    types::{SequenceLoadState, SequenceLoadStateDetail, SequenceLoadStateMap},
+    types::{
+        SequenceLoadState, SequenceLoadStateDetail, SequenceLoadStateMap, SequenceTitlesLoadState,
+    },
 };
 use crate::app::types::Sequence;
-use luda_editor_rpc::Socket;
-use namui::{render, Entity, Namui, NamuiImpl, Wh, XywhRect};
+use luda_editor_rpc::{DirentFileType, Socket};
+use namui::{render, Entity, Namui, NamuiImpl, Wh};
 use std::{collections::HashMap, time::Duration};
 use wasm_bindgen_futures::spawn_local;
-mod button_background;
 mod button_text;
 mod events;
-mod load_button;
+mod list;
+mod list_item;
 mod open_button;
+mod reload_titles_button;
+mod rounded_rectangle;
 mod types;
 
+const LIST_WIDTH: f32 = 800.0;
 const BUTTON_HEIGHT: f32 = 36.0;
+const RECT_RADIUS: f32 = 4.0;
+const SPACING: f32 = 4.0;
+const MARGIN: f32 = 4.0;
 
 pub struct SequenceListProps {
-    pub xywh: XywhRect<f32>,
+    pub wh: Wh<f32>,
 }
 
 pub struct SequenceList {
     sequence_load_state_map: SequenceLoadStateMap,
+    sequence_titles_load_state: Option<SequenceTitlesLoadState>,
```

Comment:
I think it would not to be `Option` because `SequenceTitlesLoadState` has loading state, and you start load on `SequenceList::new`.

Reply:
Yes, that must be lagacy code. I'll fix them

Reply:
Fixed with 3d15814